OSDOCS-16988-f:  CQA work for INSTALL-1: Core Installation Concepts an (#114628)
openshift/openshift-docs: modules/installation-obtaining-fips-installer-oc.adoc

[id="installation-obtaining-fips-installer-oc_{context}"]
= Obtaining a FIPS-capable installation program using `oc adm extract`

[role="_abstract"]

> {product-title} requires the use of a FIPS-capable installation binary to install a cluster in FIPS mode. You can obtain this binary by extracting it from the release image by using the {oc-first}. After you have obtained the binary, you proceed with the cluster installation, replacing all instances of the `openshift-install` command with `openshift-install-fips`.

Reviewer: ```suggestion
You must get a FIPS-capable installation binary to install a {product-title} cluster in FIPS mode. Extract the binary from the release image by using the {oc-first}.  After you get the binary, you must proceed with the cluster installation, replacing all instances of the `openshift-install` command with `openshift-install-fips`.
```

Author: :heavy_check_mark:
